docs/shared/attributes.adoc

```diff
@@ -14,7 +14,7 @@
 :ProductVersion: 1.0
 
 //***NEW START***
-:Feature: RainCatcher
+:RainCatcher: RainCatcher
```

Comment:
In definition variable should never get example value. 
So typical values are projectName, label etc.

Feature seemed little bit off. 
Let's communicate this with the team directly as we need to  build new PR's with the right name.

Reply:
[13:43] 
John Stafford Suggestions for the name of this variable?
--> :RainCatcher: RainCatcher
It would be used as follows: == Introducing {RainCatcher} Security Strategies (edited)
[13:44] 
I had it originally called :Feature: RainCatcher
Any thoughts on the above?

[13:47] 
Michal Ogrodniczak RainCatcher sounds good

[13:47] 
jbriones
yeah same here. I think RainCatcher is good